bitmark-references/bitmark-node-software/bitmark-shares.md

```diff
@@ -0,0 +1,70 @@
+# Bitmark shares
+
+## Reasoning
+
+* There may be some reason to turn a property into shares in order to
+  divide the ownership.
+
+* Once a property has been divided it does not make sense to combine
```

Comment:
I'd like to see why it doesn't make sense. My attempt: because a non-fungible bitmark will lose its characteristics when it's divided so this is a irreversible action?